.github/workflows/test.yml

```diff
@@ -400,61 +403,48 @@ jobs:
           kernel-root: '.'
           kbuild-output: ${{ env.KBUILD_OUTPUT }}
           image-output: '/tmp/root.img'
-          test: run_veristat
+          test: run_veristat_kernel
```

Comment:
ok, so yes, this test is ran because it is baked in the test: https://github.com/libbpf/ci/blob/5b0b421fdc961f1ee0220ef624e71e1ebaa6dd86/prepare-rootfs/run_vmtest.sh#L25C143-L25C147

And when we finally run the init script (which in turn run the tests), it automatically pick it up here: https://github.com/kernel-patches/vmtest/blob/6766a3de584be2e69fab19e2c99c81e8a4fe0f5a/ci/vmtest/run_selftests.sh#L38-L42

I think I would prefer to remove that `test` attribute here, so no specific test is ran by default, and instead instead to run both run_veristat_meta and run_veristat_meta in "Run veristat" action.

Reply:
Should I make it part of this PR or a separate one?

Reply:
I would see this as part of this PR TBH as this is the core of what this PR is addressing.